### PR TITLE
Lang file updates

### PR DESCRIPTION
**Pull request by PeculiarGoat**
Language file fixes:

- ar_LB
- ca_ES
- da_DK
- de_DE
- ee_EE

### CODE REVIEW

**Comment by PeculiarGoat** [comment]
It will be easier to review if you hide whitespace changes.

File: site/ar_LB.all.json [hunk 1]
```diff
@@ -581,16 +581,16 @@
...
+    "shopping_error_plan_expired": { "other": "هذه الخطة لم تعد متوفرة" },
 
-    "wcag_aria_label_letterboxd": { "other": "View on Letterboxd" },
+    "wcag_aria_label_letterboxd": { "other": "عرض على Letterboxd" },
 
-    "header_banner": { "other": "ABC Cinemas – 21st  Film Festival, June 1 – 6, 2021" },
-    "app_badge_title": { "other": "Download the app to view your purchased content!" },
-    "app_badge_ios": { "other": "Download on the App Store" },
-    "app_badge_android": { "other": "Get it on Google Play" },
+    "header_banner": { "other": "سينما ABC - مهرجان الفيلم الحادي والعشرون ، 1-6 يونيو 2021" },
```
Comment (l0ud0gg) [inline site/ar_LB.all.json:589]: not really a language specific issue but seems wrong to chuck in ABC / 2021 in the default banner text. prolly should just be a generic "welcome to our festival" message? I suppose end game would be the clients will likely have their own messaging for whatever context they wanna use it for and then turn it on in admin or something so maybe its not a big deal

File: site/ee_EE.all.json [hunk 2]
```diff
@@ -1,602 +1,602 @@
...
+  "usersubscriptions_subscription_expiry": { "other": "Aegub {{.Expiry}}" },
+  "usersubscriptions_subscription_status_cancelled": { "other": "usersubscriptions_subscription_status_cancelled" },
+  "usersubscriptions_subscription_status_errored": { "other": "usersubscriptions_subscription_status_errored" },
+  "usersubscriptions_subscription_status_past_due": { "other": "usersubscriptions_subscription_status_past_due" },
+  "usersubscriptions_subscription_status_trialing": { "other": "Prooviperiood" },
+  "usersubscriptions_subscription_unsubscribe": { "other": "Tühista minu tellimus" },
+  "usersubscriptions_unsubscribe_modal_cancel": { "other": "Tühista" },
+  "usersubscriptions_unsubscribe_modal_confirm": { "other": "Jah, tühistage minu tellimus" },
+  "usersubscriptions_unsubscribe_modal_title": { "other": "Kas tühistada tellimus?" },
+  "usersubscriptions_unsubscribe_modal_body": { "other": "See tühistab paketi {{.Name}} tellimuse. Saate vaadata paketi sisu, kuni see aegub {{.Aeguv}}." },
```
Comment (l0ud0gg) [inline site/ee_EE.all.json:424]: i think you might have translated the variable here

Comment (l0ud0gg) [reply site/ee_EE.all.json:424]: {{.Aeguv}}

File: site/ee_EE.all.json [hunk 3]
```diff
@@ -1,602 +1,602 @@
...
+  "shopping_error_item_limit_exceeded": { "other": "Kahjuks on {{.Title}} välja müüdud." },
+  "availability_expires": { "other": "Aegub {{.Expiry}}" },
+
+  "shopping_card_change": { "other": "Kas krediitkaart pole õige? <a href=\"{{.SubscriptionsURL}}\">Kaardi värskendamiseks klõpsake siin.</a>" },
+
+  "user_error_too_many_pin_errors": { "other": "<p>Looks like you're having trouble.</p><p>Reset your PIN or contact us for <a href=\"{{.HelpURL}}\" target=\"_blank\">help</a>" },
+
+  "app_badge_title": { "other": "Ostetud sisu vaatamiseks laadige rakendus alla!" },
+  "app_badge_ios": { "other": "Laadige alla App Store'ist" },
+  "app_badge_android": { "other": "Hankige see Google Playst" },
```
Comment (l0ud0gg) [inline site/ee_EE.all.json:455]: I assume Google Play is called Google Play in all countries due to brand, prolly dont need to translate the product name here.

Comment (l0ud0gg) [reply site/ee_EE.all.json:455]: Ah i see its a suffix from the language, all good

**Review by l0ud0gg — CHANGES_REQUESTED** [review]
skim read through all the changes just to find any syntax or missing spaces / bad variables names. just found the one variable i commented on.

File: site/ar_LB.all.json (hunk 1 above)
Comment (PeculiarGoat) [reply site/ar_LB.all.json:589]: Yeah... this core translation is never going to be used... if a client has it turned on, they need their own text in there anyway.

File: site/ee_EE.all.json (hunk 2 above)
Comment (PeculiarGoat) [reply site/ee_EE.all.json:424]: Damn I thought I caught all of those.

**Review by smitsanghvi21 — APPROVED** [review]

**Pull request merged**